src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -181,8 +181,13 @@ impl LazyBatchColumnVec {
         for col in &self.columns {
             min_len = min_len.min(col.len());
         }
+        self.truncate(min_len);
+    }
+
+    /// Shortens the rows, keeping the first `len` rows and dropping the rest.
```

Comment:
What do you mean the first `len` here?

Reply:
It means we will keeping the first n rows, n equals to `len`